### ISSUE TITLE
Any callback for handle to pass the path to the handler in register_handler 

### ISSUE THREAD

**Issue by anchor00** [h2o/h2o#351]
Hello,
I want to get the "path" in the register_handler (the 2nd parameter) when the server received the event and pass it to the handler. I am thinking the handler will has switch statements and switch to different codes base on the "path". Is it a way to accept it?

static h2o_pathconf_t *register_handler(h2o_hostconf_t *hostconf, const char *path, int (*on_req)(h2o_handler_t *, h2o_req_t *))

In summary, is it a way that the handler know what is "path"?
Thanks

**Comment by kazuho** [comment]
You can access the value via `req->pathconf->path` (example: https://github.com/h2o/h2o/blob/70480563235e19f1f867101940ccccc32e973330/lib/handler/redirect.c#L42).

**Comment by anchor00** [comment]
It works great!
Thanks,
